firmware/tunerstudio/rusefi.input

```diff
@@ -1480,41 +1480,35 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 		panel = oilp_settings
 		gauge = OilPressGauge
 
-;			Engine->MAP Sensor Settings
```

Comment:
Moved from 'Engine' menu to 'Sensors'.

Reply:
Proposed new dialog layout:
![image](https://user-images.githubusercontent.com/15104768/37728466-fea62d60-2d42-11e8-94fa-c661facf8f11.png)